projects/backend/capi/articles.ts

```diff
@@ -110,8 +113,10 @@ const parseArticleResult = async (
     if (body == null) throw new Error(`Body was undefined in ${path}!`)
 
     const elements = await attempt(Promise.all(body.map(parser)))
-    if (hasFailed(elements))
+    if (hasFailed(elements)) {
+        console.error(elements)
```

Comment:
This can be removed.

Reply:
This is logging out the specific error that's occurred.